File: javascript/stack.js
//stack...working...success...
console.log("enter capacity");
const prompt=require('prompt-sync')();
const size=parseInt(prompt());
const s=[];
function add(s,x){if(s.length==size){console.log("stack full..");}
	else{s.push(x);}
}
function remove(s){
if(s.length==0){console.log("empty...");}else{
s.pop();}
}
function show(s){if (s.length==0){console.log("stack empty..");}
	else{for(var i=0;i<s.length;i++){
			console.log(s[i]);
		}}
}
console.log("original stack\n");
console.log(s);
while(true){
	console.log("1: add\t 2:remove\t 3: show stack");
	var tmp=parseInt(prompt());
	if (tmp==1){let a=parseInt(prompt('enter element to add- '));add(s,a);}
	else if (tmp==2){remove(s);}
	else if (tmp==3){show(s);}
	else{break;}
}

File: java/stack.java
import java.util.*;
/*working...success..*/
class MakeStack{
private int arr[];
private int top;
private int capacity;
MakeStack(int size){
arr=new int[size];
capacity=size;
top=-1;
}
public void push(int x){
if(isFull()){System.out.println("FULL....exiting...");System.exit(1);}
System.out.println("Inserting ..."+x);
arr[++top]=x;}
public int pop(){
if(isEmpty()){System.out.println("EMPTY..");}
return arr[top--];}
public int size(){return top+1;}
public Boolean isEmpty(){return top==-1;}
public Boolean isFull(){return top==capacity-1;}
public void printStack(){for(int i=0;i<=top;i++){System.out.print(arr[i]+" ");}System.out.println();}
public static void main(String[] s){
System.out.println("enter capacity..");
Scanner nrt=new Scanner(System.in);
int cap=nrt.nextInt(),tmp,pop;
MakeStack S=new MakeStack(cap);
while(true){
System.out.println("1: add\t 2:remove\t 3: show stack\nenter any other no. to exit..");
tmp=nrt.nextInt();
if(tmp==1){System.out.println("enter element to enter..");
pop=nrt.nextInt();
S.push(pop);}
else if(tmp==2){S.pop();}
else if(tmp==3){S.printStack();}
else{break;}}
}
}
